### PR TITLE
add warning if evaluation period is in last hours

### LINKED ISSUE #990: End of evaluation warning
When opening an evaluation questionnaire after the last day of evaluation (in the X hour period in which the evaluation is still possible), an information about the exact end of the evaluation should be shown on top of the page (or maybe in a modal).

### PR DESCRIPTION
**Pull request by maikz**
fixes #990 
+ add warning if evaluation period is in last hours
+ fix assertion and related tests

### CODE REVIEW

File: evap/student/views.py [hunk 1]
```diff
@@ -80,7 +81,11 @@ def vote(request, course_id):
             contributor_form_groups=contributor_form_groups,
             course=course,
             participants_warning=course.num_participants <= 5,
-            preview=False)
+            preview=False,
+            vote_end_datetime=course.vote_end_datetime,
```
Comment (karyon) [inline evap/student/views.py]: is that variable used in the template?

File: evap/evaluation/models.py [hunk 2]
```diff
@@ -262,7 +262,7 @@ def save(self, *args, **kw):
             self.contributions.create(contributor=None)
             del self.general_contribution  # invalidate cached property
 
-        assert self.vote_end_date >= self.vote_end_date
+        assert self.vote_end_date >= self.vote_start_datetime.date()
```
Comment (karyon) [inline evap/evaluation/models.py]: i would prefer such a change to be in a separate commit since it has little to do with the feature you implemented, but you don't need to change it for this PR.

File: evap/grades/tests.py [hunk 3]
```diff
@@ -24,8 +24,8 @@ def setUpTestData(cls):
         cls.course = mommy.make(
             Course,
             name_en="Test",
-            vote_start_datetime=datetime.now() - timedelta(10),
-            vote_end_date=datetime.now() + timedelta(10),
+            vote_start_datetime=datetime.now() - timedelta(days=10),
+            vote_end_date=date.today() + timedelta(10),
```
[the remark this answers is not in the document]
Comment (maikz) [reply evap/grades/tests.py]: days is the first argument. I think like this it's more specific

[the remark this answers is not in the document]
Comment (karyon) [reply evap/grades/tests.py]: yeah, i was suggesting adding it here, since in the line above it is specified.

File: evap/student/views.py (hunk 1 above)
Comment (karyon) [reply evap/student/views.py]: yes it is.

**Comment by karyon** [comment]
forgot to say thanks, so in case you're not there tomorrow: thanks! :)
